script/Arcueid/Day 8/08_00_ARC08_2B.txt

```diff
@@ -238,12 +238,12 @@ But he doesn't smell like a bourgeois at all, does he?!"
 [sha:d2dd8340f9831edc527793ef739d937bce5e6789]{
 -- Page 109, Offset 19940.
 -- ……なんなんだ、この状況……。
-...What on earth have I stumbled into here...
+...What on earth have I been caught up in here...
```

Comment:
Prefer original, it doesn't feel like he's really caught up in anything/ involved yet, just observing